opentelemetry-api/src/opentelemetry/correlationcontext/propagation/binaryformat.py

```diff
@@ -15,9 +15,10 @@
 import abc
 import typing
 
-from opentelemetry.distributedcontext import DistributedContext
+from opentelemetry.context import Context
 
 
+# TODO: can this be removed until it is needed?
```

Comment:
👍

AFAICT we don't even need separate text and binary "formats" with the new API.

Reply:
Removed